packages/playground/src/utils/nodeSelector.ts

```diff
@@ -43,10 +47,16 @@ export async function getLocations(status?: NodeStatus): Promise<Locations> {
         country.name = con.name;
       }
     });
-
-    if (allowedCountriesList.includes(country.name)) {
-      locations[country.region] = locations[country.region] || [];
-      locations[country.region].push(country.name);
+    if (country.region !== "unknown region") {
+      if (allowedCountriesList.includes(country.name)) {
+        locations[country.region] = locations[country.region] || [];
+        locations[country.region].push(country.name);
+      }
+    } else {
+      if (allowedCountriesList.includes(country.name)) {
+        locations[country.subregion] = locations[country.subregion] || [];
+        locations[country.subregion].push(country.name);
+      }
```

Comment:
instead of duplicating the code, can we decide if we will use the region or the sub-region and store it into a variable and use this variable?